.github/workflows/zombienet.yml

```
@@ -0,0 +1,63 @@
+name: zombienet
+run-name: Zombienet tests
+on:
+  pull_request:
+
+jobs:
+  zombienet-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+          stable: true
+          check-latest: true
+
+      - name: Set cache variables
+        id: go-cache-paths
+        run: |
+          echo "::set-output name=go-build::$(go env GOCACHE)"
+          echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+      - uses: actions/checkout@v3
+
+      - name: Go build cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go-cache-paths.outputs.go-build }}
+          key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-build
+
+      - name: Go modules cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go-cache-paths.outputs.go-mod }}
+          key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-mod
+
+      - name: Run build
+        run: |
+          make build
+          echo "$HOME/work/gossamer/gossamer/bin" >> $GITHUB_PATH
+
+      - name: Install Zombienet
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            wget -P $HOME/.local/bin/ https://github.com/paritytech/zombienet/releases/download/v1.3.41/zombienet-linux-x64
+            mv $HOME/.local/bin/zombienet-linux-x64 $HOME/.local/bin/zombienet
```

```suggestion
            wget -O $HOME/.local/bin/zombienet $HOME/.local/bin/ https://github.com/paritytech/zombienet/releases/download/v1.3.41/zombienet-linux-x64
```

---

when I changed this there is error: `/home/runner/.local/bin/zombienet: No such file or directory`, so I'm not changing this.

---

Ah dang yeah the directory `$HOME/.local/bin/` doesn't exist. Maybe then

```suggestion
            mkdir -p $HOME/.local/bin
            wget -O $HOME/.local/bin/zombienet $HOME/.local/bin/ https://github.com/paritytech/zombienet/releases/download/v1.3.41/zombienet-linux-x64
```

Or why not directly to `/usr/local/bin` which exists:

```suggestion
            wget -O /usr/local/bin/zombienet https://github.com/paritytech/zombienet/releases/download/v1.3.41/zombienet-linux-x64
```

---

Nice, updated.